src/orderbook-basics.yaml

```diff
@@ -12,23 +12,371 @@ publishStatus: Live
+            key: D
+
+  - uuid: 6fc6f39c-81a8-4796-aade-bfff10267e15
+    name: Advantages of CLOB
+    content: |
+      Some of advantages of CLOB model are:
+
+      * Users can execute orders using smart contracts in Serum's fully decentralized, on-chain order book while keeping pricing and order size flexibility.
+
+      * Similar to a standard CLOB, users have the option to specify the price, quantity, and direction of their transactions. This technique provides high liquidity by matching orders based on price, time, and priority.
+
+      * Serum enhances composability as it can be used by other protocols in the area to bootstrap liquidity and provide matching services.
+
+      * The fast throughput and low transaction costs of the Solana network assist the Serum order book in minimising capital inefficiencies and liquidity segmentation.
+
+      * Serum's CLOB is built to be asset agnostic and has additional strong features like cross-chain swaps. It may create an order book that corresponds to any trading product offered by Solana, including options and futures (e.g. ETH and BTC).
+
+      * Due to Serum's flexible protocol, a wide range of participants and applications can share middleware in one place. This is made possible by the backend matching engine's versatility, which allows it to be extended to almost any financial or non-financial item.
+      
+      * Other applications can connect to Serum and leverage its infrastructure to build their own projects thanks to on-chain CLOB, such as a trading application that utilizes Serum's liquidity.   
     stepItems: []
+
+  - uuid: 512e6dc7-5919-43e4-bfd2-40b26ebb2d81
+    name: Evaluation
+    content: |
+
+    stepItems:
+      - uuid: 53c58fcf-8101-48f8-9681-899a4b173a0b
+        content: CLOB enhances the liquidity by specifying ____ of transaction.
+        answerKeys:
+          - A
+          - B
+          - C
+        type: MultipleChoice
+        choices:
+          - content: Price 
+            key: A
+          - content: Quantity
+            key: B
+          - content: Direction
+            key: C
+          - content: Transaction Time
+            key: D
+
+      - uuid: 02db90e6-1301-4bdf-b04e-b97eaafa22e1
+        content: How does Solana empower Serum CLOB?
+        answerKeys:
+          - D
+        type: SingleChoice
+        choices:
+          - content: Low Transaction Cost
+            key: A
+          - content: Minimising capital inefficiencies
+            key: B
+          - content: Less Liquidity Segmentation
+            key: C
+          - content: All of the above
+            key: D
+      
+      - uuid: 5f5b0575-7d48-4365-a523-2033a64fabb4
+        content: What do you mean by cross-chain swaps?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: Swapping within a blockchain ecosystem
+            key: A
+          - content: Swapping across different Cryptos using smart contracts
```

Comment:
What do we mean by `different Cryptos` here?

Reply:
Different tokens